pkg/plugins/wrapper.go

```diff
@@ -0,0 +1,36 @@
+package plugins
+
+import (
+	"github.com/hashicorp/go-plugin"
+	"github.com/treeverse/lakefs/pkg/logging"
+)
+
+// Wrapper wraps the go-plugin client (which is in fact a plugin controller) and the GRPC client.
+// It's used as a convenience struct to call the different types of plugins, and to be included as a subtype of specific
+// plugin client implementations
+type Wrapper struct {
+	GRPCClient *plugin.ClientProtocol
+	Client     *plugin.Client
+	Log        logging.Logger
+}
+
+// InitClient initializes the inner GRPC client to communicate to the
+func (c *Wrapper) InitClient() error {
+	if c.GRPCClient != nil {
+		return nil
+	}
```

Comment:
- Think it will be better to return an error as it will look like a bug calling init more than once. Or remove the check
- Wrote the first item as you also do not protect the Destroy (calling more than once) - why like that? ;)